convert2rhel/unit_tests/utils_test.py

```diff
@@ -280,11 +364,11 @@ def test_run_cmd_in_pty_check_for_deprecated_string():
 
 TERMINAL_SIZE_SCRIPT = """
 def terminal_size():
-    import fcntl, termios, struct, sys
-    h, w, hp, wp = struct.unpack('HHHH',
-        fcntl.ioctl(sys.stdout.fileno(), termios.TIOCGWINSZ,
-        struct.pack('HHHH', 0, 0, 0, 0)))
-    return w, h
+	import fcntl, termios, struct, sys
+	h, w, hp, wp = struct.unpack('HHHH',
+		fcntl.ioctl(sys.stdout.fileno(), termios.TIOCGWINSZ,
+		struct.pack('HHHH', 0, 0, 0, 0)))
+	return w, h
```

Comment:
@Andrew-ang9, can you rollback this change here? They are unrelated to the scope of your PR. 

```suggestion
def terminal_size():
    import fcntl, termios, struct, sys
    h, w, hp, wp = struct.unpack('HHHH',
        fcntl.ioctl(sys.stdout.fileno(), termios.TIOCGWINSZ,
        struct.pack('HHHH', 0, 0, 0, 0)))
    return w, h

print(terminal_size()[0])

```

Reply:
Yeah I can get this taken out of the PR.